ui/components/app/gas-customization/gas-modal-page-container/gas-modal-page-container.container.js

```diff
@@ -138,12 +138,10 @@ const mapStateToProps = (state, ownProps) => {
       });
   const isGasEstimate = getIsGasEstimatesFetched(state);
 
-  let customPriceIsSafe;
+  let customPriceIsSafe = true;
```

Comment:
Default to true if on a standard testnet or in the fallback scenario where the call for gas prices has failed (i.e. `isGasEstimate` is false). This was the intent of https://github.com/MetaMask/metamask-extension/pull/11111